api/src/main/java/com/epam/pipeline/controller/user/UserController.java

```diff
@@ -274,4 +292,47 @@ public Result deleteUser(@PathVariable Long id) {
     public Result<List<String>> findGroups(@RequestParam String prefix) {
         return Result.success(userApiService.findGroups(prefix));
     }
+
+    @PostMapping(value = "/group/block")
+    @ResponseBody
+    @ApiOperation(
+            value = "Creates the block status of a group.",
+            notes = "Creates the block status of a group. " +
+                    "If the group is blocked, none of its users can't access their accounts.",
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(
+            value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)
+            })
+    public Result<GroupStatus> createGroupBlockingStatus(@PathVariable final String groupName,
```

Comment:
Change `blockStatus` to `@RequestParam`, add `groupName` to url path -> `/group/{groupName}/block`